.travis.yml

```diff
@@ -25,7 +25,7 @@ install: skip
 # Run maven build in quiet mode to avoid log file limit in Travis CI.
 # Only errors and Maven version information will be printed in the CI logs.
 script:
-  - mvn clean install -B -V | grep -e "\[ERROR\]" -e Maven -e "// Start Running" -e "Building"; test ${PIPESTATUS[0]} -eq 0;
+  - mvn clean install -PRelease -B -V | grep -e "\[ERROR\]" -e Maven -e "// Start Running" -e "Building"; test ${PIPESTATUS[0]} -eq 0;
```

Comment:
this builds wasm for every PR.
it'll be nice if we can trigger this profile manually for wasm updated PRs.

Reply:
In ubuntu based machines wasm build in few minutes . So no issue I giess

Reply:
ah ok. otherwise we could do something like below for github actions build (by introducing a new github label). keeping the travis as it is.
https://stackoverflow.com/questions/62325286/run-github-actions-when-pull-requests-have-a-specific-label